packages/manager/modules/sms/src/sms/batches/create/routing.js

```diff
@@ -0,0 +1,170 @@
+          TucToast.success($translate.instant('sms_batches_create_success'));
+        }),
+      getSendersHref: /* @ngInject */ ($state) => () =>
+        $state.href('sms.service.senders'),
+      getOrderHref: /* @ngInject */ ($state) => () =>
+        $state.href('sms.service.order'),
+      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+      goToComposeTip: /* @ngInject */ ($state) => () =>
+        $state.go('sms.service.batches.create.compose-tips'),
+      goToPhonebooksContacts: /* @ngInject */ ($state) => (model) =>
+        $state.go('sms.service.batches.create.phonebooks-contacts', {
+          model,
+        }),
+      goToReceiversChoice: /* @ngInject */ ($state) => (model) =>
+        $state.go('sms.service.batches.create.receivers-choice', {
+          model,
+        }),
+      goToSizeTip: /* @ngInject */ ($state) => () =>
+        $state.go('sms.service.batches.create.size-tips'),
+      groupBySenderType: /* @ngInject */ ($translate) => (sender) =>
+        $translate.instant(
+          `sms_sender_optgroup_label_${
+            sender.sender === SMS_COMPOSE.shortNumber ? 'virtual' : sender.type
+          }`,
+        ),
+      phonebooks: /* @ngInject */ (iceberg, serviceName) =>
+        iceberg(`/sms/${serviceName}/phonebooks`)
+          .query()
+          .expand('CachedObjectList-Pages')
+          .execute()
+          .$promise.then(({ data }) => data),
+      receivers: /* @ngInject */ (iceberg, serviceName) =>
+        iceberg(`/sms/${serviceName}/receivers`)
+          .query()
+          .expand('CachedObjectList-Pages')
+          .execute()
+          .$promise.then(({ data }) => data),
+      receiversUrl: () => SMS_CONSTANTS.SMS_URL.guides.receivers,
+      schema: /* @ngInject */ ($http) =>
+        $http.get('/sms.json').then(({ data: schema }) => schema),
+      selectedReceivers: /* @ngInject */ ($transition$) =>
+        $transition$.params().selectedReceivers,
+      senders: /* @ngInject */ ($q, $translate, iceberg, serviceName) =>
+        iceberg(`/sms/${serviceName}/senders`)
+          .query()
+          .expand('CachedObjectList-Pages')
+          .execute()
+          .$promise.then(({ data }) => data)
+          .then((sendersDetails) =>
+            sendersDetails.map((sender) => ({
+              ...sender,
+              label: sender.sender,
+            })),
+          )
+          .then((sendersDetails) => [
+            {
+              sender: SMS_COMPOSE.shortNumber,
+              label: $translate.instant('sms_sender_option_short_number'),
+            },
+            ...sendersDetails,
+          ]),
+      smsClasses: /* @ngInject */ (schema) =>
+        schema.models['sms.ClassEnum'].enum.map((smsClass) => ({
+          value: smsClass,
+          label: smsClass.replace('_', ' '),
+        })),
+    },
+    translations: {
+      value: ['.'],
+      format: 'json',
+    },
+  });
+
+  $stateProvider.state('sms.service.batches.create.receivers-choice', {
```

Comment:
Could those be in separate routing declaration ? As they are supposed to be part of a different module